core/src/main/scala/akka/persistence/cassandra/query/EventsByTagStage.scala

```diff
@@ -269,7 +269,10 @@ import scala.compat.java8.FutureConverters._
                     continue()
                   } else if (settings.eventsByTagEventualConsistency < settings.refreshInterval) {
                     // No point scheduling for EC if the QueryPoll will come in beforehand
-                    scheduleOnce(TagNotification, settings.eventsByTagEventualConsistency)
+                    // No point in scheduling more frequently than once every 10 ms
+                    scheduleOnce(
+                      TagNotification(System.currentTimeMillis() / 10),
```

Comment:
where is this resolution used

Reply:
it's just a way to create "unique" timer keys to avoid cancelling previous, unique per 10ms

Reply:
ah gotcha